packages/expo-cli/src/commands/run/android/runAndroid.ts

```diff
@@ -146,6 +153,38 @@ export async function runAndroidActionAsync(projectRoot: string, options: Option
   }
 }
 
+function track(projectRoot: string, exp: ExpoConfig) {
+  UnifiedAnalytics.logEvent('dev client run command', {
+    status: 'started',
+    platform: 'android',
+    ...getDevClientProperties(projectRoot, exp),
+  });
+  StatusEventEmitter.once('bundleBuildFinish', () => {
+    // Send the 'bundle ready' event once the JS has been built.
+    UnifiedAnalytics.logEvent('dev client run command', {
```

Comment:
I know the rest of the cli uses hardcoded strings but it might be nice to transition to some enum of event types like in other codebases 🙂

Reply:
I think I actually prefer this way. I was trying to use search to find where a certain event is triggered in EAS CLI codebase and it was made harder by the constants and additional abstractions.

More thorough TypeScript type for `logEvent` might be useful, though, and would not lose the searchability.